common/pkg/k8s/client/create.go

```diff
@@ -0,0 +1,30 @@
+// Copyright 2024 Nutanix. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package client
+
+import (
+	"context"
+	"fmt"
+
+	ctrlclient "sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+func Create(
```

Comment:
Why do we need this new Create func over [Apply](https://github.com/nutanix-cloud-native/cluster-api-runtime-extensions-nutanix/blob/4b5cfde67d21d16542408082ae62f15dea86cd25/common/pkg/k8s/client/apply.go#L22)?

Reply:
This was to ensure that we dont update the secret with new randomly generated key. Once I added check for existing secret, `create` or `apply` would not matter, however I kept there to be clear that we do not update this secret in CAREN once it is generated.